ui/component/publishFile/view.jsx

```diff
@@ -216,6 +216,8 @@ function PublishFile(props: Props) {
       } else {
         updateFileInfo(0, file.size, isVideo);
       }
+    } else {
+      updateFileInfo(0, file.size, isVideo);
```

Comment:
I'm not sure if `file.size` will cause any side effects for non-videos (should I just zero it?), but I believe the previous code just uses whatever the previous value was, so most likely it doesn't matter.

Reply:
I think it's fine, as long as `file` isn't undefined. Which it doesn't look like it is